torch/_dynamo/symbolic_convert.py

```diff
@@ -208,6 +209,9 @@
 compare_op_handlers["not in"] = lambda tx, args, _: handle_not(
     tx, [handle_contains(tx, [*reversed(args)], {})], {}
 )
+latest_bytecode_queue: deque[str] = deque(
```

Comment:
This global is causing your 13/20 issue I believe. Can you move this queue to be a member of InstructionTranslator?

Reply:
Fixed. Thank you!